src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -78,6 +78,29 @@ public T WithInstanceDicoveryMetadata(string instanceDiscoveryJson)
             }
         }
 
+        /// <summary>
+        /// Allows developers to configure their own valid authorities. A Uri that returns a response similar to https://aka.ms/aad-instance-discovery 
```

Comment:
configure. Wouldn't it rather be describe?
Maybe we also can write that this URL is used when the metadata is not at the expected location authority/discovery/instance?

Reply:
@jmprieur  : This is not when the metadata is not at the expected location. This is to override default behavior. 
The description is rather: Lets an organization setup their own instance for discovery, which enables better caching for microservice/service environments.